backend/polls/views.py

```diff
@@ -201,25 +182,21 @@ def statistics(self, *args, **kwargs):
             time_to_next_poll = (tomorrow - datetime.now()).seconds
 
             # Setting up cache to avoid recomputing the dicts
-            cache.set("participations", participations_by_user, time_to_next_poll)
-            cache.set("victories_ranking", victories_by_user, time_to_next_poll)
-            cache.set("defeats_ranking", defeats_by_user, time_to_next_poll)
+            cache.set("poll_leaderboard", poll_leaderboard, time_to_next_poll)
 
         sorted_users_participations = sorted(
```

Comment:
Could we avoid to  re-sort it every time?

Reply:
I think it's not a big deal because there are at most ~5k users so it should take around 1ms to sort the list.
What would you suggest?

Reply:
Ok let's leave it that way